src/disco/fakeData.js

```diff
@@ -7,19 +12,21 @@ export default {
     slug: 'japanese-tattoo',
     sub_heading: null,
     type: 'Theme',
-    url: 'https://addons-dev.allizom.org/en-US/firefox/addon/japanese-tattoo/',
-    headerURL: 'https://addons-dev-cdn.allizom.org/user-media/addons/18781/personare.jpg?1239806327',
-    footerURL: 'https://addons-dev-cdn.allizom.org/user-media/addons/18781/persona2re.jpg?1239806327',
+    url: `${amoHost}/en-US/firefox/addon/japanese-tattoo/`,
+    themeURL: `${amoCDN}/user-media/addons/18781/preview_large.jpg?1239806327`,
```

Comment:
Should these names be like `themeUrl`? I'm assuming the API will return them as `theme_url` and [camelcase will make that `themeUrl`](https://tonicdev.com/5728c8c86ad20d1100700899/573b7355f544491100596c27).